README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
FWIW I don't love using `0` to mean no timeout. That's the behavior of Ruby's `Timeout::timeout`. But in the `select` syscall a `NULL` timeout means no timeout, and a `0` timeout means "time out immediately":
> If both fields of the timeval structure are zero, then select() returns immediately.
(from [man page](http://man7.org/linux/man-pages/man2/select.2.html))

I can imagine wanting to time out immediately for testing purposes.

It's probably more important to be consistent with `Timeout::timeout` than with `select`, but I'd prefer a more explicit value (like `:disabled`) in addition to `0`.